google-cloud-pubsub/acceptance/pubsub/schema_test.rb

```diff
@@ -17,7 +17,7 @@
 
 # This test is a ruby version of gcloud-node's pubsub test.
 
-describe Google::Cloud::PubSub::Schema, :pubsub do
+describe Google::Cloud::PubSub::V1::Schema, :pubsub do
```

Comment:
We probably should delete this test file altogether. The test is testing the old `Project#create_schema` method, and others, that we deleted. The only reason the test isn't failing is that it's skipped, referencing #20925 (which we should also now close as obsolete.)